packages/core/review-workflows/shared/contracts/review-workflows.ts

```diff
@@ -115,7 +116,9 @@ namespace GetAll {
 namespace Update {
   export interface Request {
     body: {
-      data: Partial<Workflow>;
+      data: Partial<Omit<Workflow, 'stageRequiredForPublish'>> & {
+        stageRequiredForPublish?: Stage['name'] | null;
+      };
     };
```

Comment:
Just to clarify, how does this differ from the original `Partial<Workflow>`?

Reply:
On Workflow interface, stageRequiredForPublish is of type Stage, this is okay because we return that and this field is a relation. 

But for Create/Update endpoins, we don't receive the complete Stage, only the name, so I omit that type and add this stageRequiredForPublish that is only name